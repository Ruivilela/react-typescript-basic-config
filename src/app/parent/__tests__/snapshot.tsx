import React from 'react';
import renderer from 'react-test-renderer';
import Parent from '..';

describe('Parent', () => {
  it('should render', () => {
    const component = renderer.create(<Parent/>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
