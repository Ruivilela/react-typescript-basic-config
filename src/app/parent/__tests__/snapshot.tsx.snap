// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Parent should render 1`] = `
<div>
  <input
    onChange={[Function]}
    value=""
  />
</div>
`;
